### PR TITLE
[hotfix] [docs] Fix document address does not match the actual web page.

### PR DESCRIPTION
**Pull request by ZhijieYang**
## What is the purpose of the change

Fix document address does not match the actual web page.

## Brief change log

- Fix the bug from `docs/content.zh/docs/dev/datastream/fault-tolerance/serialization/third_party_serializers.md`
- modify the link form `docs/content.zh/docs/dev/datastream/fault-tolerance/serialization/types_serialization.md`

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: no
  - The serializers: no
  - The runtime per-record code paths (performance sensitive): no
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: no
  - The S3 file system connector: no

## Documentation

  - Does this pull request introduce a new feature? no
  - If yes, how is the feature documented? no

### CODE REVIEW

**Comment by ZhijieYang** [comment]
@wuchong  Hi, can you review it? thanks

**Comment by ZhijieYang** [comment]
@RocMarshal Hi, can you review it? this error can stop build the doc

**Comment by RocMarshal** [comment]
@ZhijieYang Thanks for your contribution. I'll advance it ASAP.

File: docs/content.zh/docs/dev/datastream/fault-tolerance/serialization/types_serialization.md [hunk 1]
```diff
@@ -259,7 +259,7 @@ The most frequent issues where users need to interact with Flink's data type han
   by itself. Not all types are seamlessly handled by Kryo (and thus by Flink). For example, many Google Guava collection types do not work well
   by default. The solution is to register additional serializers for the types that cause problems.
   Call `.getConfig().addDefaultKryoSerializer(clazz, serializer)` on the `StreamExecutionEnvironment`.
-  Additional Kryo serializers are available in many libraries. See [Custom Serializers]({{< ref "docs/dev/datastream/fault-tolerance/serialization/custom_serializers" >}}) for more details on working with custom serializers.
+  Additional Kryo serializers are available in many libraries. See [3rd party serializer]({{< ref "docs/dev/datastream/fault-tolerance/serialization/third_party_serializers" >}}) for more details on working with external serializers.
```
Comment (RocMarshal) [inline docs/content.zh/docs/dev/datastream/fault-tolerance/serialization/types_serialization.md:262]:
```suggestion
  Additional Kryo serializers are available in many libraries. See [3rd party serializer]({{< ref "docs/dev/datastream/fault-tolerance/serialization/third_party_serializers" >}}) for more details on working with custom serializers.
```

File: docs/content.zh/docs/dev/datastream/fault-tolerance/serialization/third_party_serializers.md [hunk 2]
```diff
@@ -3,8 +3,8 @@ title: 自定义序列化器
 weight: 11
```
Comment (RocMarshal) [inline docs/content.zh/docs/dev/datastream/fault-tolerance/serialization/third_party_serializers.md:3]: Maybe we should keep the titile consistent with English part.

**Review by RocMarshal — COMMENTED** [review]
@ZhijieYang Good catch. left minor comments. PTAL.

File: docs/content.zh/docs/dev/datastream/fault-tolerance/serialization/third_party_serializers.md (hunk 2 above)
Comment (ZhijieYang) [reply docs/content.zh/docs/dev/datastream/fault-tolerance/serialization/third_party_serializers.md:3]: As you wish

File: docs/content.zh/docs/dev/datastream/fault-tolerance/serialization/types_serialization.md (hunk 1 above)
Comment (ZhijieYang) [reply docs/content.zh/docs/dev/datastream/fault-tolerance/serialization/types_serialization.md:262]: As you wish, I will both change it to `custom serializers` in the English and Chinese Version

File: docs/content.zh/docs/dev/datastream/fault-tolerance/serialization/third_party_serializers.md (hunk 2 above)
Comment (RocMarshal) [reply docs/content.zh/docs/dev/datastream/fault-tolerance/serialization/third_party_serializers.md:3]: @klion26  Hi, What's your opinion ?

Comment (ZhijieYang) [reply docs/content.zh/docs/dev/datastream/fault-tolerance/serialization/third_party_serializers.md:3]: How about we put this issue on hold and focus on fixing the errors in the documentation first, I noticed that this page is not translated, later I want to create a JIRA to fix the translation of the whole `docs/content.zh/docs/dev/datastream/fault-tolerance/serialization/third_party_serializers.md` page, we can discuss that in the next pr, this time leave it as it is for now.

Comment (RocMarshal) [reply docs/content.zh/docs/dev/datastream/fault-tolerance/serialization/third_party_serializers.md:3]: @ZhijieYang OK. Hi, @wuchong , please let us know what do you think of it.

**Comment by ZhijieYang** [comment]
@JingsongLi Hi, it's a long time not push on，what’s your opinion, thanks

**Pull request closed without merging**